### ISSUE TITLE
Bug ?

### ISSUE THREAD

**Issue by georgestheking** [bgolab/JustBasic#10]
Hi
I flash a new board.
? work and give the list of command
but I get "#" as prompt

Best regards
Georges

**Comment by bgolab** [comment]
To understand the issue please:
1. Could you please share the terminal log? I would like to se what happened during the booting phase, interpreter version
2. Can you execute any command like ( I mean executing from the command line): r print "hi"


I stopped working on this project for some time but will release new version soon.

**Comment by georgestheking** [comment]
Hi,

I press only "?" twice

"RESET"
JustPicoBasic v1.1B100
(C) 2021 bg
HW: RP2040
Running at 125000000Hz


Press ESC to SKIP init script!

#?

JustPicoBasic v1.1B100
(C) 2021 bg
HW: RP2040

SYSTEM: ?, [l]oad, [s]ave, [c]ode, [r]un, [n]ew, [b]ye, @n[], t0-t2

BASIC: integer, float, string, dim, rem, print, input, cls, pause, data, read, restore, if, then, else, endif, for, to, step, break, next, while, endwhile, goto, gosub, return, end, abs, sin, cos, exp, log, sqr, sgn, hex$, str$, chr$, asc, left$, mid$, right$, len, val, int, fix, and, or, not, ,, ;, +, -, *, /, %, (, ), <, <=, >, >=, =, <>, ==, !=, inkey, rnd,

HW: peek, poke, gettick, pmode, dwrite, awrite, dread, aread, lplot, ldraw, lcircle, lprint, at, lref, lcls, tone, notone, iinit, ideinit, ireadable, iwritable, iread, iwrite, uinit, udeinit, ureadable, uwritable, uread, uwrite, sinit, sdeinit, sreadable, swritable, sread, swrite, sm,

CONST: IN, OUT, PULLUP, PULLDOWN, ADC, PWM, TSENSOR, I2C0, I2C1, UART0, UART1, SPI0, SPI1, HIGH, LOW,

#

**Comment by georgestheking** [comment]
interpreter does not work 👍 

#print "hi"
print "hi"
#
#
#
#

**Comment by georgestheking** [comment]
Sorry It work 
I use RUN in place of r

Best regards
Georges

**Comment by bgolab** [comment]
Yes, the differentiation between calculator-like mode and programming mode might be misleading.

I wrote several interpreters and tried a few approaches like explicit 'user' mode with '>' prompt for ad-hoc programming but here decided to use 'r' command prefix... Every approach has its pluses and minuses.
